### PR TITLE
fix: Make Ticket Inspection window a Paginated Table

### PR DESCRIPTION
**Pull request by HappyNTH**
Fixes overflow problem on mobile with the new ticket inspection window

### CODE REVIEW

File: pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue [hunk 1]
```diff
@@ -16,34 +16,45 @@
...
-          <table-head-item>Seat Group</table-head-item>
-          <table-head-item>Concession Type</table-head-item>
-          <table-head-item>Checked In</table-head-item>
-          <table-head-item v-if="anyTicketsChecked"
-            >Checked In By</table-head-item
+      <UiTablesPaginatedTable
+        v-model:offset="ticketInspectionOffset"
+        :items="booking.tickets"
+        :max-per-page="10"
+        :page-info="{}"
```
Comment (atoff) [inline pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue:23]: If you aren't using these features I'd just remove them

File: pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue [hunk 2]
```diff
@@ -80,6 +91,8 @@ import BookingStatusEnum from '~~/enums/PayableStatusEnum';
 import { dateFormat } from '@/utils/datetime';
 import { AdminBookingDetailDocument } from '~~/graphql/codegen/operations';
 
+const ticketInspectionOffset = ref(0);
```
Comment (atoff) [inline pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue:94]: Along with this - you shouldn't need it

File: pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue (hunk 1 above)
Comment (HappyNTH) [reply pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue:23]: `offset`, `items` and `page-info` all seem required by the component, and the page crashes out without them: I can only remove `max-per-page` without issue...

Comment (atoff) [reply pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue:23]: Ok fair enough - this component is meant to help you implement pagination very easily. Probably won't run into that issue here so I don't really mind. Just hard-code the offset as 0 (better not to register a reactive variable if we don't have to for performance!): `v-model:offset` -> `:offset="0"`

**Review by atoff — APPROVED** [review]

**Pull request merged**